src/app/beer_garden/events/processors.py

```diff
@@ -125,16 +128,47 @@ def register(self, processor, manage: bool = True):
 class HttpEventProcessor(QueueListener):
     """Publish events using an EasyClient"""
 
-    def __init__(self, easy_client=None, black_list=None, **kwargs):
+    def __init__(
+        self, easy_client=None, black_list=None, reconnect_action=None, **kwargs
+    ):
         super().__init__(**kwargs)
 
         self._ez_client = easy_client
         self._black_list = black_list or []
+        self._reconnect_action = reconnect_action
 
     def process(self, event: Event):
+
         try:
             if event.name not in self._black_list:
                 event.garden = beer_garden.config.get("garden.name")
                 self._ez_client.publish_event(event)
         except Exception as ex:
             logger.exception(f"Error publishing EasyClient event: {ex}")
+
+            self.reconnect()
+
+    def reconnect(self):
+
```

Comment:
Should add an warning-level log message indicating that we're in this reconnect state.

Reply:
Done